Library/Formula/bugzilla.rb

```diff
@@ -0,0 +1,22 @@
+class Bugzilla < Formula
+  desc "Bugzilla is server software designed to help you manage software development."
+  homepage "https://www.bugzilla.org"
+  url "https://ftp.mozilla.org/pub/mozilla.org/webtools/bugzilla-5.0.1.tar.gz"
+  version "5.0.1"
```

Comment:
Homebrew should be able to scan the version from the URL, so this shouldn't be necessary.